apps/web/src/app/(dashboard)/documents/delete-document-dialog.tsx

```diff
@@ -83,47 +88,82 @@ export const DeleteDocumentDialog = ({
     <Dialog open={open} onOpenChange={(value) => !isLoading && onOpenChange(value)}>
       <DialogContent>
         <DialogHeader>
-          <DialogTitle>Are you sure you want to delete "{documentTitle}"?</DialogTitle>
+          <DialogTitle>Are you sure?</DialogTitle>
 
           <DialogDescription>
-            Please note that this action is irreversible. Once confirmed, your document will be
-            permanently deleted.
+            You are about to {canManageDocument ? 'delete' : 'hide'}{' '}
```

Comment:
Awesome work!

My only feedback so far is that using 2 different terminologies "hide" and "delete" might confuse people. Wouldn't be better to stay consistent and use the same terminology? Curious to see what you and others think.

Reply:
I used hide because the documents aren't actually deleted, which can cause a different kind of confusion

The reason it works for gmail and other services is generally because the item is deleted after X days

Whereas our system will only hide it

Reply:
Yeah, makes sense. Well, let's ship it and see the feedback. 🙏